lib/ui.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require_relative 'game.rb'
+require_relative 'board.rb'
+require_relative 'player.rb'
+
+class UI
```

Comment:
I suggest that instead of making this a class let's just make this a module. Anyway, it just contains auxiliary methods and not holding vital data in contrast to Player, Board and Game